Unreliable crash when using filesystem plugin with path

# Description
When using `2ms filesystem --path somedir/` on a non empty dir, sometimes there's a crash (possible data-race).

# Steps to reproduce
1. Create a directory `mkdir testdir`
2. Create a file in the dir `echo "someinfo" > testdir/testfile.txt`
3. Run `2ms filesystem --path testdir`

# Expected Behavior
Expecting the command to succeed and report no secrets.

# Current Behavior
Sometimes ( around 1 / 3 times ) the program panics with this error stack:
```
panic: sync: WaitGroup is reused before previous Wait has returned

goroutine 1 [running]:
sync.(*WaitGroup).Wait(0x0?)
	/usr/local/go/src/sync/waitgroup.go:118 +0x7f
github.com/checkmarx/2ms/cmd.postRun(0xc00012a600?, {0xa79f14?, 0x2?, 0x2?})
	/home/<redacted>/Downloads/2ms/cmd/main.go:171 +0x3a
github.com/spf13/cobra.(*Command).execute(0xc00012a600, {0xc0001242c0, 0x2, 0x2})
	/home/<redacted>/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:936 +0xa2b
github.com/spf13/cobra.(*Command).ExecuteC(0xef3ce0)
	/home/<redacted>/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/<redacted>/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/checkmarx/2ms/cmd.Execute()
	/home/<redacted>/Downloads/2ms/cmd/main.go:107 +0x5e5
main.main()
	/home/<redacted>/Downloads/2ms/main.go:22 +0x1fa
exit status 2
```
I removed my information from the stack trace and replaced with <redacted>

--- a/plugins/filesystem.go
+++ b/plugins/filesystem.go
@@ -77,6 +77,7 @@ func (p *FileSystemPlugin) getItems(items chan Item, errs chan error, wg *sync.W
 	for _, filePath := range fileList {
 		wg.Add(1)
 		go func(filePath string) {
+			defer wg.Done()
 			actualFile, err := p.getItem(wg, filePath)
 			if err != nil {
 				errs <- err
@@ -88,7 +89,6 @@ func (p *FileSystemPlugin) getItems(items chan Item, errs chan error, wg *sync.W
 }
 
 func (p *FileSystemPlugin) getItem(wg *sync.WaitGroup, filePath string) (*Item, error) {
-	defer wg.Done()
 	b, err := os.ReadFile(filePath)
 	if err != nil {
 		return nil, err

